### PR TITLE
Pass meta to level fn

### PR DESCRIPTION
**Pull request by ecoutu**
Passing `meta` object constructed for bunyan log function to `levelFn`. This provides more information for determining log level to use.

### CODE REVIEW

**Comment by ecoutu** [comment]
I can rebase this branch for you if you are interested in merging this pull request.

**Comment by ecoutu** [comment]
Ok, rebased master, should be clear to merge.

File: test/bunyan.test.js [hunk 1]
```diff
@@ -268,6 +268,31 @@ describe('bunyan-logger', function() {
           done();
         });
   });
+
+  it('test options.levelFn', function (done) {
+    var app = express();
+    var output = st();
+    app.use(bunyanLogger({
+      stream: output,
+      levelFn: function (status, err, meta) {
```
Comment (marcbachmann) [inline test/bunyan.test.js]: That api is a bit strange.
Later we can probably move that to the first argument as `{status, meta}`.
But that means we have to make a major version bump.

Comment (marcbachmann) [inline test/bunyan.test.js]: I'll merge this for now.

**Pull request merged**